.gitignore

```diff
@@ -69,3 +69,6 @@ docs/
 # android keystores
 android/app/signing.properties
 *.keystore
+
+# vim temp files
+*.swp
```

Comment:
@hawkrives that was in reference to you offering instructions on how you got spacemacs working. I make plenty use of emacs keybindings, and I do end up using vim for the majority of things.